src/flows/components/FlowsIndex.tsx

```diff
@@ -12,6 +12,13 @@ import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
 import {PROJECT_NAME_PLURAL} from 'src/flows'
 
 const FlowsIndex = () => {
+  const handleFeedbackClick = (): void => {
+    window.open(
+      'https://w2.influxdata.com/feedback-notebooks-early-access/',
```

Comment:
is this something we want to exist elsewhere as a const? Does that matter?

Reply:
Probs more semantic to just use an anchor tag, but ran into some styling jank and this was the path of least resistance. This button will only exist so long as we're in early access so probs not something that needs more permanence

Reply:
Gotcha. Totally chill with leaving it there if it's gonna get cut out later